app.js

```diff
@@ -0,0 +1,77 @@
+
+console.log("Hello, World!"); //->English
+console.log("Silav, Cihan!");//->Kurdish
+console.log("Hoi, Wereld!"); //->Dutch
+
+console.log("I'm awesome!");
+
+let x =
+console.log('I think the value of x is bigger than 5');
+console.log("The value of x is 6");
+let x = 6;
+console.log(I think the value of x is 6);
+console.log(x);
+
+let y ="Sivan";
+console.log("I think the value of y is a name");
+console.log(y);
+const y = "Hassam";
+console.log("There will be an error because now I have
+             the same variable with different values.");
+console.log(y);
+
+const z = 7.25;
+console.log(z);
+const a = z;
+console.log(a);
+var round =Math.round(7.25);
+console.log(round)
+
+let arrays =
+console.log("I think variable arrays is related to animals");
+console.log(arrays);
+let myFavoriteAnimals = [cat, dog, sheep];
+console.log(myFavoriteAnimals);
+let favoriteAnimalOfDaan = "baby pig";
+const animals = "myFavoriteAnimals" + "" + "favoriteAnimalOfDaan";
+console.log(animals);
+
+let myString = "this is a test";
+console.log(myString);
+console.log(myString ["length"]);
+
+let num = 11;
+let address = "Huygenhoekring";
+let supermarkets = ["Lidl", "Dekka", "Jumbo"];
+let booleans = true, false, true;
+console.log(num);
+console.log(address);
+console.log(supermarkets);
+console.log(booleans);
+console.log(I think the type of num is number,
+            type of address is string,  
+            type of supermarkets is array and
+            type of booleans is boolean.);
+console.log(typeof num);
+console.log(typeof supermarkets);
+console.log(typeof booleans);
+console.log(typeof address);
+
+if (num != address) {
+    console.log("num is a number and address is a string");
+}
+
+let s = 7;
+let v = 3;
+let h = s % 3;
+console.log(s % h);
+console.log(s % v);
+console.log(v % h);
+
+const mix = [5, "Shvan"];
+console.log(mix);
+
+var maxNumber = Math.pow(6/0);
+
+if (maxNumber === Infinity)
+console.log(0 / maxNumber);

```

Comment:
I would recommend adding brackets here, so like this:
```
if (maxNumber === Infinity) {
  console.log(0 / maxNumber);
}
```

It makes it easier to avoid mistakes

Reply:
Changed to: var maxNumber = Math.pow(6/0);

if (maxNumber === Infinity){
console.log(0 / maxNumber);
}